shims/spark32/src/main/scala/io/glutenproject/execution/GenerateTreeStringShim.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.execution
+
+import org.apache.spark.sql.execution.SparkPlan
+
+trait GenerateTreeStringShim extends SparkPlan {
```

Comment:
@PHILO-HE @ulysses-you 

If we extend `TreeNode`, we would also need to specify [BaseType](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala#L69). However, these operations have already been specified when `SparkPlan `extends `TreeNode`, so there is no need to do it again here. I am more inclined to extend `UnaryExecNode`, because `WhostageTransformer ` also extends the `UnaryExecNode ` abstract class from Spark.